cli/src/cli_util.rs

```diff
@@ -1883,6 +1890,14 @@ fn load_revset_aliases(
             }
         }
     }
+
+    if !aliases_map.contains_symbol(ALIAS_TRUNK) {
+        aliases_map.insert(
+            ALIAS_TRUNK,
+            "latest((present(main) | present(master) | present(root)) & remote_branches())",
```

Comment:
Have you considered the alternative of putting this as a revset alias in a TOML file? The user can then just redefine it if they want.

You could put this in `https://github.com/martinvonz/jj/blob/main/cli/src/config/misc.toml` or, perhaps better, create `https://github.com/martinvonz/jj/blob/main/cli/src/config/aliases.toml` for both aliases and revset aliases.

Reply:
Amazing, I didn't realise that was there, definitely a cleaner/better approach.